src/Native/ObjWriter/objwriter.h

```diff
@@ -50,7 +50,7 @@ enum class RelocType {
 
 class ObjectWriter {
 public:
-  bool Init(StringRef FunctionName);
+  bool Init(StringRef FunctionName, const char* tripleName = nullptr);
```

Comment:
Could you please change the argument names to start with upper case in the C++ to match the LLVM coding conventions that the ObjectWriter follows?